unattended_installer/install_functions/checks.sh

```diff
@@ -278,21 +278,21 @@ function checks_previousCertificate() {
     fi
 
     if [ -n "${indxname}" ]; then
-        if ! $(tar -tf "${tar_file}" | grep -q "${indxname}".pem) || ! $(tar -tf "${tar_file}" | grep -q "${indxname}"-key.pem); then
+        if ! $(tar -tf wazuh-install-files.tar wazuh-install-files/${indxname}.pem >/dev/null 2>&1) || ! $(tar -tf wazuh-install-files.tar wazuh-install-files/${indxname}-key.pem >/dev/null 2>&1); then
```

Comment:
Should we use the`"` for `${indxname}` variable?

Reply:
Done in [c07a505](https://github.com/wazuh/wazuh-packages/pull/1488/commits/c07a5050dae2ca4194318489caea40608d04e38a)